bylaws.md

```diff
@@ -1,5 +1,197 @@
+
+Financial matters within CCS include:
+
+- **[Capital Contributions]**: Member funds invested into the cooperative for initial setup or growth. Used for acquiring assets, covering expenses, etc.
+- **[Profit distribution]**: The allocation of net income to members based on patronage credits. In the form of cash, or reinvested into the co-op.
+- **[Operating expenses]**: Costs for running the co-op day-to-day. Includes rent, utilities, salaries, maintenance.
+- **[Contract distributions]**: Payments or benefits members receive from specific contracts or projects. May be fixed or percentage-based.
+
+## 📝 Amendments
+
+This document may be amended or revised from time to time. Amendments require the approval of 75% of the members and will be communicated through official channels.
+
+## ⚖️ Dispute Resolution
+
+In the event of a dispute among members, CCS promotes the use of mediation or arbitration to resolve conflicts in a fair and efficient manner.
+
+## 📚 Governing Law
+
+This legal document is governed by the laws of Minnesota. Any legal proceedings related to this document shall take place in the courts of Minnesota.
+
+## Member Voting 
+
+The voting power of the members shall be equal.
+
+Whenever members are disqualified from voting on any matter, they shall not be counted for the determination of a quorum or the required vote to approve an action.
+
+There shall be no voting by proxy.
+
+There shall be no cumulative voting.
+
+## Member Resignation
+
+A member shall have the right to resign as a member of the cooperative by filing with the secretary of the cooperative a written notice of resignation.  The resignation shall become effective immediately without any action on the part of the cooperative.  An individual will not be allowed to work for the cooperative for one month following voluntary termination of membership unless her or his membership is reinstated by the cooperative.  Resignation shall not relieve the resigning member from any obligation for charges incurred, dues, assessments, or fees, and this section shall not diminish any right of the cooperative to enforce any such obligation or obtain damages for its breach.
+
+##  Removal of Members
+
+No member may be expelled or suspended, and no membership or memberships may be terminated or suspended, except according to procedures satisfying the requirements of this section.
+
+A member may for any lawful reason be expelled from the cooperative by a vote of seventy-five percent (75%) or greater of the members at a duly called meeting at which a quorum is present.
+
+The member must be given 15 days’ prior notice of the expulsion, suspension, or termination and the reasons therefor and the member shall have an opportunity to be heard, orally or in writing, not less than five days before the effective date of the expulsion, suspension, or termination by a person or body authorized to decide that the proposed expulsion, termination, or suspension not take place.
```

Comment:
```suggestion
The member must be given 15 days’ prior notice of the expulsion, suspension, or termination and the reasons therefore and the member shall have an opportunity to be heard, orally or in writing, not less than five days before the effective date of the expulsion, suspension, or termination by a person or body authorized to decide that the proposed expulsion, termination, or suspension not take place.
```

Interesting I looked up if there's a difference between _therefor_ and _therefore_ and there **is.**   "Therefor" (without the e) generally means "in return for that" while "Therefore" (with an e) roughly translates to "because."  

Anyway I imagine we want **therefore.**